src/App.vue

```diff
@@ -31,6 +31,7 @@ watch(
   },
   { immediate: true }
 )
+statePersistorLayersService.bootstrap()
```

Comment:
Why do you need to call it a second time here?

Reply:
This is a mistake ^^...